src/main/java/com/troubleshooting/model/ConjurLogModel.java

```diff
@@ -0,0 +1,28 @@
+package com.troubleshooting.model;
+
+// Data object for a Conjur log
```

Comment:
Class documentation looks as follows:
/**
* Foo bar
*/
Java SDK contains a documentation generator that outputs HTML